src/main/java/net/geant/nmaas/portal/api/market/UsersController.java

```diff
@@ -7,13 +7,7 @@
 import net.geant.nmaas.notifications.MailAttributes;
 import net.geant.nmaas.notifications.NotificationEvent;
 import net.geant.nmaas.notifications.templates.MailType;
-import net.geant.nmaas.portal.api.domain.PasswordChange;
-import net.geant.nmaas.portal.api.domain.PasswordReset;
-import net.geant.nmaas.portal.api.domain.UserBase;
-import net.geant.nmaas.portal.api.domain.UserRequest;
-import net.geant.nmaas.portal.api.domain.UserRoleView;
-import net.geant.nmaas.portal.api.domain.UserView;
-import net.geant.nmaas.portal.api.domain.UserViewMinimal;
+import net.geant.nmaas.portal.api.domain.*;
```

Comment:
Please check this rule: https://rules.sonarsource.com/java/RSPEC-2208
In general start imports should not be used. You can alter IDEA configuration so that star imports are not added automatically.